fmn/rules/utils.py

```diff
@@ -198,6 +299,65 @@ def _get_pkgdb2_packages_for(config, username, flags):
     return dict(packages)
 
 
+def _get_pagure_packages_for(config, username, flags):
+    """
+    Get the packages a user is associated with from pagure.
+
+    Args:
+        config (dict): The application configuration.
+        username (str): The FAS username to fetch the packages for.
+        flags (list): The type of relationship the user should have to the
+            package (e.g. "watch", "point of contact", etc.).
+    Returns:
+        dict:  A `dict` mapping namespaces to sets of package names.
+    """
+    log.debug("Requesting pagure packages for user %r" % username)
+
+    # See https://github.com/fedora-infra/fmn/issues/209
+    # which depends on https://pagure.io/pagure/issue/2421
+    valid_flags = ['point of contact', 'co-maintained']#, 'watch']
+
+    bogus = set(flags) - set(valid_flags)
+    if bogus:
+        log.error("%r are not valid owner flags for %r." % (bogus, username))
+        # ... but, proceed.
+
+    valid = set(flags) & set(valid_flags)
+    if not valid:
+        log.error("No valid owner flags by which to query.")
+        return dict()
+
+    base = config.get('fmn.rules.utils.pagure_api_url',
+                      'https://src.fedoraproject.org/pagure/api')
+    url = base + '/0/projects'
```

Comment:
FYI - I go on PTO end of day Friday.  I granted you both access to my fork in case you want to push additional patches there to update this PR:  https://github.com/ralphbean/fmn

I'll try to fix before I leave!